smoke-test-node.bash

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env bash
```

Comment:
I'm wondering why we need this script (instead of calling the test-node.bash script to start up the services).